### ISSUE TITLE
clickhouse-server/store/shadow: no such file or directory

### ISSUE THREAD

**Issue by everythings-gonna-be-alright** [Altinity/clickhouse-backup#124]
Backup doesn't work. I run next command and got error:
```
~# clickhouse-backup create --table='mydatabase.blablabla' my_backup
2020/10/27 15:20:44 Create backup 'my_backup'
2020/10/27 15:20:45 Freeze `mydatabase`.`blablabla`
2020/10/27 15:20:45 Copy metadata
2020/10/27 15:20:45   Done.
2020/10/27 15:20:45 Move shadow
2020/10/27 15:20:45 open /var/lib/clickhouse-server/store/shadow: no such file or directory
```

Some information about my cluster:
```
~# clickhouse-server --version
ClickHouse server version 20.10.2.20 (official build).
```

```
~# clickhouse-backup --version
Version:	 0.6.0
Git Commit:	 7d7df1e36575f0d94d330c7bfe00aef7a2100276
Build Date:	 2020-10-02
```

```
SHOW CREATE DATABASE mydatabase

┌─statement──────────────────────────────┐
│ CREATE DATABASE mydatabase
ENGINE = Ordinary │
└────────────────────────────────────────┘

1 rows in set. Elapsed: 0.003 sec. 
```

```
SHOW CREATE DATABASE system

┌─statement─────────────────────────────┐
│ CREATE DATABASE system
ENGINE = Atomic │
└───────────────────────────────────────┘

1 rows in set. Elapsed: 0.003 sec.
```

**Comment by AlexAkulov** [comment]
Backup available only for MergeTree family tables.

**Comment by everythings-gonna-be-alright** [comment]
@AlexAkulov Sorry, i forgot about this part.
In this case i try to backup exactly tables with `ReplicatedMergeTree` type.
And it works well with ClickHouse 20.7.2.30
I think it happen because of new engine Atomic. It's used by default for system table since ClickHouse 20.8

```
│ CREATE TABLE mydatabase.blablabla
(
(
    `id` Int64,
    `test1` Nullable(String),
    `test2` Nullable(Int64)
)
ENGINE = ReplicatedMergeTree('/my-clickhouse/tables/{shard}/blablabla', '{replica}')
ORDER BY id
SETTINGS index_granularity = 8192 │
```

**Comment by hohaftf** [comment]
And Atomic is the default database engine since ClickHouse 20.10
So, I guess this will be a common case

**Comment by hohaftf** [comment]
@everythings-gonna-be-alright 
I've succeeded by specifying config file explicitly and setting data path in it

**Comment by everythings-gonna-be-alright** [comment]
@hohaftf 
Yes, confirm. Everything works  fine after setting data path.
Thanks a lot 🙃

```
data_path: '/var/lib/clickhouse-server'
```

**Comment by AlexAkulov** [comment]
Yes, I reproduced issue with Atomic Engine and data_path error in 20.10. It will be fixed in the next version.

**Comment by hohaftf** [comment]
The CREATE command doesn't create metadata folder in backup. And the RESTORE command fails with 
>2020/10/29 12:09:55 stat /var/lib/clickhouse/backup/2020-10-28T11-47-44/metadata: no such file or directory

**Comment by AlexAkulov** [comment]
Error "open /var/lib/clickhouse-server/store/shadow: no such file or directory" on backup creation fixed in [v0.6.1](https://github.com/AlexAkulov/clickhouse-backup/releases/tag/v0.6.1).
I created a new issue for Atomic database support https://github.com/AlexAkulov/clickhouse-backup/issues/126
